src/main.c

```diff
@@ -57,9 +57,12 @@ static void draw_row_callback(GContext *ctx, Layer *cell_layer, MenuIndex *cell_
   }
 }
 
+#ifdef PBL_ROUND
```

Comment:
Either use a shim or ifdef on SDK version (with comments), assuming the reason you're doing this is because the various symbols don't exist on 2.x